ideascube/serveradmin/models.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+
+from ideascube.models import JSONField
+
+
+class Setting(models.Model):
+    class Meta:
+        ordering = ['-date']
+
+    namespace = models.CharField(max_length=40)
+    key = models.CharField(max_length=40)
+    value = JSONField()
+    actor = models.ForeignKey(settings.AUTH_USER_MODEL)
+    date = models.DateTimeField(auto_now=True)
+
+    def as_boolean(self):
```

Comment:
I agree on class methods over manager, and on the API you suggest, raising only when no default is given.
My only concern is that we *may* need to have `None` as default value.
I don't like it that much, but an alternative option would be to have `**kwargs` and to check for `default` being actually passed.